executor/executor.go

```diff
@@ -81,6 +81,7 @@ const (
 	codeErrBuildExec    terror.ErrCode = 9
 	codeBatchInsertFail terror.ErrCode = 10
 	// MySQL error code
+
```

Comment:
Here is the downside of `goword`, when we need to comment several fields, goword makes it harder to read.